src/utils/scripts/csvToJSON.js

```diff
@@ -0,0 +1,15 @@
+import { writeFileSync } from "fs";
+import csv from 'csvtojson';
+
+const dataName = "Data"
+const csvFilePath='data.csv';
+
+csv()
+.fromFile(csvFilePath)
+.then((jsonObj)=>{
+    console.log(jsonObj);
+    let json = JSON.stringify(jsonObj);
+    writeFileSync('output.json', `{"${dataName}":${json}}`);
```

Comment:
Can we manipulate the json to look like the firestore data?
Right now, from what I understand of the code, it will only write an array of objects. Let's try to create a generic function that will create an object of the following structure:
```
Question: {
  text: string;
  answer: [
    {
      answer: string;
      points: number;
    }
  ]
}
```